src/Feature/Keyboard/Feature_Keyboard.re

```diff
@@ -0,0 +1,77 @@
+open Oni_Core;
```

Comment:
I had to create this separate feature project because `Feature_MenuBar` depends on `Feature_Input`, so putting this there created a cyclic dependency. Maybe it makes sense to refactor some of the stuff #3546 on top of this if it gets merged?